src/vs/workbench/api/node/extHostTerminalService.ts

```diff
@@ -12,16 +12,17 @@ export class ExtHostTerminal implements vscode.Terminal {
 
 	public _name: string;
 	public _shellPath: string;
```

Comment:
`_shellPath` and `_shellArgs` can safely be removed as they're not exposed like name